packages/editor/src/components/rich-text/format-toolbar/index.js

```diff
@@ -11,7 +18,16 @@ const FormatToolbar = ( { controls } ) => {
 				{ controls.map( ( format ) =>
 					<Slot name={ `RichText.ToolbarControls.${ format }` } key={ format } />
 				) }
-				<Slot name="RichText.ToolbarControls" />
+				<Slot name="RichText.ToolbarControls">
+					{ ( fills ) => fills.length &&
+						<DropdownMenu
```

Comment:
In terms of design, any button added to the block toolbar should ideally be wrapped in `<Toolbar>`. There are many examples of this in the code base and it gives the right aria-roles, design...

Reply:
I think we are using:
```js
{ ( fills ) => ! isEmpty( fills ) && (
```
in other places. I don't remember why. Ideally, we should be using `fills.length`.

Reply:
Okey, I double checked our code and it's safe to use `fills.length`. We should default to it moving on 👍 

Intrernally we use `map` from lodash which ensures it returns an array:
https://github.com/WordPress/gutenberg/blob/master/packages/components/src/slot-fill/slot.js#L66

![screen shot 2019-03-08 at 11 14 51](https://user-images.githubusercontent.com/699132/54022822-ab36bd00-4193-11e9-87da-cd993f77f294.png)